test/graphql_practice_web/schema/mutations/upload_test.exs

```diff
@@ -21,152 +21,152 @@ defmodule GraphqlPracticeWeb.Schema.Mutations.UploadTest do
-                 variables: %{
-                   "title" => create_title,
-                   "description" => create_description,
-                   "user_id" => user_id
-                 }
-               )
-
-      assert [
-               %{message: "description: can't be blank", path: ["createUpload"]},
-               %{message: "title: can't be blank", path: ["createUpload"]}
-             ] = errors
-    end
-  end
-
-  @update_upload_doc """
-   mutation UpdateUpload($id: Id!, $title: String, $description: String){
-    updateUpload(id: $id, title: $title, description: $description){
-     id
-     title
-     description
-     user {
-       name
-       id
-     }
-  }
-  }
-  """
-
-  describe "@update_upload" do
-    setup [:user, :upload]
-
-    test "Updates an upload when valid params are provided", %{upload: upload} do
-      update_title = "updated title"
-      update_description = "updated description"
-      id = upload.id
-
-      assert {:ok, %{data: data}} =
-               Absinthe.run(@update_upload_doc, Schema,
-                 variables: %{
-                   "title" => update_title,
-                   "description" => update_description,
-                   "id" => id
-                 }
-               )
-
-      assert %{"updateUpload" => upload} = data
-
-      assert %{
-               "title" => ^update_title,
-               "description" => ^update_description
-             } = upload
-    end
-
-    test "Returns errors when ID is not found", %{upload: upload} do
-      update_title = "updated title"
-      update_description = "updated description"
-      non_existent_id = upload.id + 1
-
-      assert {:ok, %{errors: errors}} =
-               Absinthe.run(@update_upload_doc, Schema,
-                 variables: %{
-                   "title" => update_title,
-                   "description" => update_description,
-                   "id" => non_existent_id
-                 }
-               )
-
-      assert [%{message: "Upload not found!", path: ["updateUpload"]}] = errors
-    end
-  end
-
-  @delete_upload_doc """
-   mutation DeleteUpload($id: Id!){
-    deleteUpload(id: $id){
-     id
-     title
-     description
-     user {
-       name
-       id
-     }
-  }
-  }
-  """
-
-  describe "@delete_upload" do
-    setup [:user, :upload]
-
-    test "Deletes an upload when valid id is provided", %{upload: upload} do
-      id = upload.id
-
-      assert {:ok, %{data: data}} =
-               Absinthe.run(@delete_upload_doc, Schema,
-                 variables: %{
-                   "id" => id
-                 }
-               )
-
-      string_id = to_string(id)
-
-      assert %{"deleteUpload" => deleted_upload} = data
-
-      assert %{"description" => "some description", "id" => ^string_id, "title" => "some title"} =
-               deleted_upload
-
-      assert_raise Ecto.NoResultsError, fn -> Content.get_upload!(id) end
-    end
-  end
+  # describe "@create_upload" do
```

Comment:
Why is this commented out?

Reply:
Ahhhh! Because I forgot that they were commented out.....